stable/traefik/values.yaml

```diff
@@ -357,3 +357,18 @@ tracing:
   #   localAgentHostPort: "127.0.0.1:8126"
   #   debug: false
   #   globalTag: ""
+
+## Create HorizontalPodAutoscaler object.
+##
+autoscaling: {}
```

Comment:
Can this be commented out?

Reply:
Sure, given that it doesn't have default values, it makes sense.